scss/_patterns_switch.scss

```diff
@@ -0,0 +1,61 @@
+@import 'settings';
+
+@mixin vf-p-switch {
+  .p-switch {
+    align-items: stretch;
+    border: 0;
+    display: inline-flex;
+    font-size: .875rem;
+    height: 2rem;
+    padding: initial;
+    position: relative;
+    width: 4rem;
+
+    &:focus {
+      outline: 1px solid $color-link;
+      outline-offset: 3px;
+    }
+
+    :first-child,
```

Comment:
Yeah this is valid you need to quickly look at what is written again. I've changed your example to make it work https://codepen.io/richmccartney/pen/BdPQGP?editors=1100